cl_ui3d2d_extras.lua

```diff
@@ -96,10 +96,12 @@ do --3d2d VGUI Drawing
             checkHover(panel, cursorX, cursorY)
         end
 
+        panel:SetPaintedManually(false)
```

Comment:
These two calls to SetPaintedManually() can be trimmed down to just one on the first paint.